bam2depth.c

```diff
@@ -154,7 +168,7 @@ int main_depth(int argc, char *argv[])
         n = argc - optind; // the number of BAMs on the command line
     data = calloc(n, sizeof(aux_t*)); // data[i] for the i-th input
     reg_tid = 0; beg = 0; end = INT_MAX;  // set the default region
-    if(print_header) fputs("#CHROM\tPOS",stdout);
+    if( print_header ) fputs("#CHROM\tPOS", file_out);
```

Comment:
Look at the existing code nearby in this function. What I was asking for is `if (print_header)`, not this `if( foo )` abomination.

Reply:
Sadly the `if ( foo )` notation is commonly used too in our code. Bah.

Some counts, as I was curious:

`if(x)   ` 16
`if( x )  ` 3
`if (x)  ` 4873
`if ( x )` 1165

The One True Way is fortunately winning the battle. :-)

Reply:
Yes. It's Petr's preferred style, and I was never able to get him to compromise and use the local style when he was working on One True Way-formatted files — so the infection has made inroads outwith bcftools.